functions.php

```diff
@@ -37,19 +37,42 @@ function extractDataAndDisplay(array $data):string
     return $text;
 }
 
+function executeFormCheckAndDataInsert(array $get): string
+{
+    $message = '';
+    if (empty($_GET)) {
+        $message = '' ;
+    } else {
+        $title = $_GET['title'];
+        $artist_name = $_GET['artist_name'];
+        $type = $_GET['type'];
+        $description = $_GET['description'];
+        $message .= isFormCorrect($title, $artist_name, $type, $description);
+        if ($message === '') {
+            $db = dbConnect();
+            $message .= isInDataBase($db,$title);
+            if ($message === '') {
+                $db = dbConnect();
```

Comment:
You're already instantiating the `$db` on line 52, use that one instead of getting a new connection.